src/components/alerts/alertBanner.tsx

```diff
@@ -28,6 +28,6 @@ type ContainerProps = {
     mode: NonNullable<AlertBannerProps['mode']>;
 };
 
-const Container = styled.div.attrs(({ mode }: ContainerProps) => ({
+const Container = styled.div.attrs<ContainerProps>(({ mode }) => ({
```

Comment:
Hey, I'm curious. Why the switch in what's being typed?

Reply:
After updating `styled-components` to v6 the previous way of specifying the prop types was throwing a type error, I think something changed on the types definition of `styled-components`